app/controllers/api/v0/service_plans_controller.rb

```diff
@@ -16,7 +16,7 @@ def order
 
         render :json => {:task_id => task.id}
       rescue ActiveRecord::RecordNotFound
-        head :bad_request
+        head :not_found
```

Comment:
@Hyperkid123 I think we should change this back, then the CI will pass.

@bdunne does the IPP says we should return 400 for not found btw?

Reply:
oh right forgot to discard that change